packages/react/src/components/Notification/Notification.js

```diff
@@ -16,8 +16,27 @@ import {
   WarningFilled20,
 } from '@carbon/icons-react';
 
+import Button from '../Button';
+
 const { prefix } = settings;
 
+export const NotificationActionButton = ({ actions }) => {
```

Comment:
Can we define `className`, `children` (instead of `label`) and `onClick` props instead of just `actions`?

Reply:
Yeah I’m not sure why we’d want to lump all of the props together as a single object. Updating a property on the object wouldn’t force a re-render due to shallow compare. So a dynamic onclick might not function as expected. Just splitting the action up into Its own props for the button is the way to go I think.

Reply:
Thanks @asudoh!

Thank you for your additional comment @vpicone -- if you are confused about this usage, please feel to ping me in Slack & I can point you to some examples of similar button series we have in GHE.